docs/Project-Configuration/Evergreen-Data-for-Analytics.md

```diff
@@ -58,6 +58,151 @@ Daily aggregated statistics for test executions run in Evergreen. Test stats are
+| host\_id                       | VARCHAR   | ID of the host that ran this task.
+| host\_create\_details          | VARCHAR   | Information about why host.create failed for this task.
+| pod\_id                        | VARCHAR   | ID of the pod that ran this task.
+| container                      | VARCHAR   | Name of the container configuration for running a container task.
+| container\_options             | ROW       | container_options row that configures the container to run the task.
+| container\_allocated           | BOOLEAN   | Whether a container has been allocated for the task.
+| container\_allocation_attempts | BIGINT    | Number of attempts made to allocate a container.
+| results\_service               | VARCHAR   | Name of the service that stores this task's test results.
+| results\_failed                | BOOLEAN   | Whether any of the task's tests have failed.
+| must\_have_results             | BOOLEAN   | Whether the task will be failed on account of no test results.
+| is\_github\_check              | BOOLEAN   | Whether the task should be considered for mainline github checks.
+| can\_reset                     | BOOLEAN   | Whether the task has successfully archived and is in a valid state to be reset.
+| reset\_when\_finished          | BOOLEAN   | Indicates that a task should be reset once it is finished running.
+| reset\_failed\_when\_finished  | BOOLEAN   | That a display task only restart failed tasks when it's restarted.
+| generate\_task                 | BOOLEAN   | Indicates that the task generates other tasks.
+| generated\_tasks               | BOOLEAN   | Indicates that the task has already generated other tasks.
+| generated\_by                  | VARCHAR   | ID of the task that generated this task.
+| generated\_json                | VARCHAR   | Configuration information to update the project YAML for generate.tasks.
+| generate\_error                | VARCHAR   | Error encountered while generating tasks.
+| generated\_task\s_to\_stepback | ROW       | Override activation for these generated tasks, because of stepback.
+| trigger\_id                    | VARCHAR   | ID of an upstream entity that triggered this task.
+| trigger\_type                  | VARCHAR   | Type of an upstream build that triggered this task.
+| trigger\_event                 | VARCHAR   | ID of the event that triggered this task.
+| commit\_queue\_merge           | BOOLEAN   | If this task is a commit queue merge task.
+| can\_sync                      | BOOLEAN   | If the task can sync when finished running.
+| sync\_at\_end\_opts            | ROW       | sync_at_end_opts row configuring task sync.
+| status                         | VARCHAR   | Task status.
+| details                        | ROW       | Task end details row.
+| oom\_killer                    | ROW       | oom_killer detector row.
+| modules                        | ROW       | Modules applied for the task.
```

Comment:
It looks like the row description is missing from below.

Reply:
fixed